ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -49,40 +42,14 @@ pub async fn test_unpopulated_offer(peertest: &Peertest, target: &Client) {
     );
 }
 
-pub async fn test_unpopulated_offer_fails_with_missing_content(
-    peertest: &Peertest,
-    target: &Client,
-) {
-    info!("Testing Unpopulated OFFER/ACCEPT flow with missing content");
-
-    let (content_key, _content_value) = fixture_header_by_hash();
-
-    // validate that wire offer fails if content not available locally
-    match target
-        .wire_offer(
-            Enr::from_str(&peertest.bootnode.enr.to_base64()).unwrap(),
-            vec![content_key.clone()],
-        )
-        .await
-    {
-        Ok(_) => panic!("Unpopulated offer should have failed"),
-        Err(e) => {
-            assert!(e
-                .to_string()
-                .contains("Content key not found in local store"));
-        }
-    }
-}
-
 pub async fn test_populated_offer(peertest: &Peertest, target: &Client) {
```

Comment:
Along with @morph-dev comment above, you can update the name of this test & log to just `test_offer` ... and remove any reference to `unpopulated`/`populated` in other test names / logs / comments in this file